src/signals/incident-management/containers/IncidentDetail/index.js

```diff
@@ -33,225 +20,283 @@ import AttachmentViewer from './components/AttachmentViewer';
 import StatusForm from './components/StatusForm';
 import Detail from './components/Detail';
 import LocationPreview from './components/LocationPreview';
+import CloseButton from './components/CloseButton';
+import IncidentDetailContext from './context';
+
+const StyledRow = styled(Row)`
+  position: relative;
+`;
 
 const DetailContainer = styled(Column)`
   flex-direction: column;
   position: relative;
+  z-index: 1;
+  justify-content: flex-start;
 `;
 
-export class IncidentDetail extends React.Component {
-  // eslint-disable-line react/prefer-stateless-function
-  constructor(props) {
-    super(props);
+const reducer = (state, action) => {
```

Comment:
There is still a lint warning here (Expected to return a value at the end of arrow function)